modepy/nodes.py

```diff
@@ -1,3 +1,35 @@
+# {{{ docstring
+
+"""
+Generic Shape-Based Interface
+-----------------------------
+
+.. currentmodule:: modepy
+
+.. autofunction:: node_count_for_shape
+.. autofunction:: node_tuples_for_shape
+.. autofunction:: equispaced_nodes_for_shape
+.. autofunction:: edge_clustered_nodes_for_shape
```

Comment:
Don't feel strongly about it at all, but `biunit_nodes_for_shape`?

Reply:
You mean `biunit_edge_clustered_...`? or `biunit_equispaced...`? Or drop one of them and only offer `biunit_` for edge-clustered?

Reply:
I was thinking just `edge_clustered_nodes_for_shape` -> `biunit_nodes_for_shape`, mostly inspired by seeing `biunit_vertices_for_shape`.

Reply:
I guess I was thinking of it as having a function `biunit_nodes_for_shape` that just does a "give me some not horrible nodes of some sort", so it doesn't matter if it's a bit vague.